src/openforms/registrations/contrib/objects_api/templatetags/objects_api.py

```diff
@@ -0,0 +1,18 @@
+import json
+
+from django import template
+from django.utils.safestring import SafeString
+
+from openforms.formio.rendering.structured import render_json
+
+register = template.Library()
+
+
+@register.simple_tag(takes_context=True)
+def json_summary(context):
+    submission = context.get("_submission")
+    if submission:
```

Comment:
Maybe nicer to do:

```
if not submission:
    return {}
```
And then the rest (so that the 'important' part is not nested).